compose/testcontainers/compose/__init__.py

```diff
@@ -65,14 +71,33 @@ def __enter__(self) -> "DockerCompose":
     def __exit__(self, exc_type, exc_val, exc_tb) -> None:
         self.stop()
 
+    def base_docker_compose(self):
```

Comment:
Let's make this a private method starting with `_` because we don't need to expose it to the user.